Dockerfile

```
@@ -32,7 +32,8 @@ COPY --from=xx / /
 
 RUN apt-get update \
     && apt-get install -y clang lld cmake protobuf-compiler jq \
-    && rustup component add rustfmt
+    && rustup component add rustfmt \
+    && rm -rf /var/lib/apt/lists/*
```

Since this is an intermediate container, removing apt sources is not needed here as they're dropped anyway.

I guess it's better to remove this line then.

```suggestion
    && rustup component add rustfmt
```